app/jobs/update_user_in_account_api_job.rb

```diff
@@ -0,0 +1,15 @@
+class UpdateUserInAccountApiJob < ApplicationJob
+  queue_as :default
```

Comment:
Do we need to specify this if it's the default?

Reply:
I think, confusingly, this is actually *not* the default - "default" is what we've called the queue used for everything other than email jobs.

Reply:
Oh my, that's confusing. 😅